compiler/damlc/tests/daml-test-files/Examples.daml

```diff
@@ -36,8 +36,8 @@ import Trace()
 import PreludeTest()
 import Text()
 import Optional()
-import NextMap()
-import NextSet()
+-- import NextMap()
```

Comment:
can we just delete it? Or if it actually matters, add the deprecation to the expected warnings. We probably need to split it in 2, one with `SINCE-LF` and one with `UNTIIL-LF` since only one triggers the warning.

Reply:
I don't think it matters, I can just delete these lines.